Cargo.toml

```diff
@@ -23,13 +23,13 @@ repository = "https://github.com/awslabs/avp-local-agent"
 
 [dependencies]
 # Main definitions for a Cedar-based authorization agent
-cedar-local-agent = "1.0"
+cedar-local-agent = { git="http://github.com/cedar-policy/cedar-local-agent", branch = "release/2.0.0" }
```

Comment:
Kind reminder to update this to 2.0 later

Reply:
Yea, the build will fail until it's updated. I'm just using this branch as a test until 2.0 is out.